Using PhotoSwipe 5.3 with TypeScript

Thank you @dimsemenov for this great software and @irudoy for the Typescript definitions.

Today I have tried to use PhotoSwipe 5.2 (and 5.3) via Typescript. In VC I can simply use the following code:

```ts
import PhotoSwipe from "photoswipe";
//...
new PhotoSwipe({});
//...
```
No errors are displayed. But trying to package it, using typescript 4.7.4, I get the following message:
```
ERROR in ./src/uz-gallery.ts 7:37-58
Module not found: Error: Package path . is not exported from package /ts/node_modules/photoswipe (see exports field in /ts/node_modules/photoswipe/package.json)
 @ ./src/app.ts 11:21-44
```

Do you have any hint how to solve that issue?

I have this problem in TypeScript:
> TS2307: Cannot find module 'photoswipe/lightbox' or its corresponding type declarations.

![image](https://user-images.githubusercontent.com/25438601/182814736-f8fdeb84-cc50-427b-ba3e-0c1c1dedcc62.png)

--- a/package.json
+++ b/package.json
@@ -8,11 +8,11 @@
   "exports": {
     ".": {
       "types": "./dist/types/photoswipe.d.ts",
-      "import": "./dist/photoswipe.esm.js"
+      "default": "./dist/photoswipe.esm.js"
     },
     "./lightbox": {
       "types": "./dist/types/lightbox/lightbox.d.ts",
-      "import": "./dist/photoswipe-lightbox.esm.js"
+      "default": "./dist/photoswipe-lightbox.esm.js"
     },
     "./dist/photoswipe.css": "./dist/photoswipe.css",
     "./photoswipe.css": "./dist/photoswipe.css",

